internal/sync/raftpb/rpc.pb.go

```diff
@@ -80,7 +80,9 @@ func init() {
 	proto.RegisterType((*InternalRaftRequest)(nil), "dkv.raftpb.InternalRaftRequest")
 }
 
-func init() { proto.RegisterFile("internal/sync/raftpb/rpc.proto", fileDescriptor_3c180fc0df45f14f) }
+func init() {
+	proto.RegisterFile("internal/sync/raftpb/rpc.proto", fileDescriptor_3c180fc0df45f14f)
```

Comment:
Since this appears to be only a formatting change, please revert this file.

Reply:
With new RPC changes for distributedService, there has been now change in proto file, and this file had to be regenerated by protoc.